### DISCUSSION TITLE
ExtensionPay eligibility outside Chrome Store

### DISCUSSION THREAD

**Discussion by 1200dG** [Glench/ExtPay#189] [General]
Hello everyone,

I was wondering, in the case I build an extension that gets rejected by the Chrome Store, can I still distribute it using the ExtensionPay features? I have seen that there's this variable InstallType, which is triggered to be non development, once it is published to Chrome Store. But, I cannot trigger it, since I don't have it publish on Chrome Store. That said, how can I overcome this situation?

Any ideas? @Glench, your input would be extremely helpful!

Thanks everyone,
João Santos

**Comment by Glench** [comment]
Hey João, I answered this in an email to you several weeks ago but here's my answer in case anyone else finds this.

ExtPay only runs in development mode if the extension is installed unpacked. Otherwise it runs in production mode. So if you distribute it as a CRX then it should run ok. But you can test it yourself to see for sure.
